### ISSUE TITLE
Files Imported from/to Access database not placed in specified vba_folder

### ISSUE THREAD

**Issue by fabrizionastri** [Aeraphe/xvba-repo#16]
**Issue Description:**
When using the "XVBA - Live Server VBA" to edit VBA code for an Access database, the files aren't placed in the directory specified in the `config.json` file: `"vba_folder": "vba-files"`. 

- The files located in `vba-files` folder are sucessfully exported to Access, and I can find them in the Access VBA Editor. 
- However, when I import the files from Access to VSCode using the "Import - VBA" button of the in the XVBA extension, the are instead located in the `C:\Users\<USERNAME>\AppData\Local\Temp` folder.

**Steps to Reproduce:**

1. Open VS Code and install "XVBA - Live Server VBA" in VSCode (v4.0.30)
2. Create a new project folder : `C:\Users\<USERNAME>\Repos\\MSAccess-XVBA-Test`
3. In the XVBA extension menu in the VSCode activity bar, click on "Boostrap XVBA Config" to create a new project, and select "Create Access Project"
![image](https://github.com/Aeraphe/xvba-repo/assets/84983143/9faa00e1-3a27-4047-896f-2259ea911aa6)
 - This will created the following files and folders in the project root: 
![image](https://github.com/Aeraphe/xvba-repo/assets/84983143/25c9c1c8-512d-4cd5-870a-0339e4696709)
 - with the following `config.json`file:
   ```json
   {
       "app_name": "XVBA",
       "description": "",
       "author": "",
       "email": "",
       "create_ate": "Mon Aug 28 2023 11:45:21 GMT+0200 (Central European Summer Time)",
       "excel_file": "index.accdb",
       "vba_folder": "vba-files",
       "ribbon_file": "customUI14",
       "ribbon_folder": "ribbons",
       "logs": "on",
       "xvba_packages": {},
       "xvba_dev_packages": {}
   }
   ```
5. Ensure the `vba-files` directory exists in the root folder of the repo and is empty.
6. Open the "index.accdb" database, and create a new modul with a VBA subroutine, for example:
  ```
  Sub TestFromAccess1()
  
  MsgBox "TestFromAccess1"
  
  End Sub
  ```
7.  In the XVBA extension menu in the VSCode activity bar, click on "Import - VBA" to import the code from Access
8. Observe that the imported files aren't placed in the `vba-files` directory.
9. Check the `logs\excel_import.log` file and find the following line: "28/08/2023 12:56:50 - DEBUG:Export VBA files from Access to C:\Users\<USERNAME>\AppData\Local\Temp\xvba_files finished. ". 
10. Go to that directory, and observe that the files are not there, but instead are located in "C:\Users\<USERNAME>\AppData\Local\Temp"
![image](https://github.com/Aeraphe/xvba-repo/assets/84983143/bd5e6830-75c9-4d81-8848-49eadb582ec7)
12. Go back to VSCode, create a new file with new sub (eg. TestFromVSCode2") and save it:
![image](https://github.com/Aeraphe/xvba-repo/assets/84983143/e7038000-f903-4455-b404-aece3b3093ba)
13.  In the XVBA extension menu in the VSCode activity bar, click on "Export VBA" to export this code to Access, and select "Build" (the other 2 options and give the same result)
![image](https://github.com/Aeraphe/xvba-repo/assets/84983143/d1fff237-c552-4421-9e1d-6f2bcc9283aa)
14. Go back to Access and open the VBA Editor (Alt+F11), and note that the previous code with TestFromAccess has disappeared, and has been replaced by the new one from VSCode.
![image](https://github.com/Aeraphe/xvba-repo/assets/84983143/662a4b63-5646-4e37-b88c-e31ef400dccd)
16. This code runs successfully when launched from within Access, but not when launched within VSCode
---

**Expected Behavior:**
The VBA files should be imported from Access to the `vba-files` directory in VSCode as specified in `config.json`. Any changes made there should then be exported back to the Access database

---

**Actual Behavior:**
The VBA files are imported in `C:\Users\<USERNAME>\AppData\Local\Temp` folder instead of `vba-files` directory.

---

**Additional Information:**

- **VS Code Version**: 1.81.1
- **XVBA Extension Version**: v4.0.30
- **Operating System**: Windows 11 Home
- I've verified that the path is correct and that the `vba-files` directory exists.
- There are no errors in the XVBA logs that indicate a path problem.

**Comment by word-x** [comment]
I got the same issue.

When I tried to import VBA code from Excel, the .bas file was not generated in vba-files folder. It  was placed in:
C:\Windows\Temp\xvba_files.

It should be generated in project_folder/vba-files.

Environment:
1.Win10, Office 2019
2.Latest VS code (I downloaded today)

![Snipaste_2023-09-12_06-26-34](https://github.com/Aeraphe/xvba-repo/assets/29170706/ed310af0-a28c-47ca-a88b-148337271b73)
![Snipaste_2023-09-12_06-27-17](https://github.com/Aeraphe/xvba-repo/assets/29170706/328fc7e4-ace8-47cb-bb22-861ef543ef34)

**Comment by word-x** [comment]
@fabrizionastri Did you fix it? It seems a bug. I've tried various versions of this extension, it has the same issue.
Very appreciated if someone could give me a hint.

**Comment by fabrizionastri** [comment]
No I did not fix it. 

@Aeraphe : Hello, are you aware of this issue ? Are you still mainting this extension ? I would appreciate any help.

**Comment by Zimbaner** [comment]
@word-x @fabrizionastri  Did you fix it? 
I've the same problem.

**Comment by alalons** [comment]
Problem solved via reverse engineering.
TLDR 
TO TEST FUNCTIONALITY Replace function with proposed one at C:\Users\YOUR_USER_NAME\AppData\Local\Temp\EXP-140880SDCEFEF-YURI-SDD-DALVA-EXFOLDERSAS-EDSDS\EXP-SDFER-FDFEWF-CANSEFEF-ACCSDEFEF-DWWCSWEF.vbs

Each time xvba is lauched via new open folder in vs code for example, this file is replaced. 
You must embed the vba modifications in the js variable to make permanent the fix. 
Remember to escape new lines with \\n and comments with \\' at least




The extension code is located at
C:\Users\YOUR_USER_NAME\\.vscode\extensions\local-smart.excel-live-server-4.0.30\dist\extension.js
this file generates some vbs scripts at C:\Users\YOUR_USER_NAME\AppData\Local\Temp\EXP-140880SDCEFEF-YURI-SDD-DALVA-EXFOLDERSAS-EDSDS
Excel related file : EXP-140880SDCEFEF-YURI-SDD-DALVA-EXFOLDERSAS-EDSDS.vbs
Access related file: EXP-SDFER-FDFEWF-CANSEFEF-ACCSDEFEF-DWWCSWEF.vbs

The acces file is created from VBSCRIPT_VBA_EXPORT_TO_ACCESS_RESOURCE variable in extension.js

The access version has two bugs:

1. It lacks a backslash to concatenate folder an file as it needs to be escaped in the VBSCRIPT_VBA_EXPORT_TO_ACCESS_RESOURCE  variable at extension.js 
cmpComponent.Export VBA_TMP_FOLDER & "\\" & fileName in the js is initialized to  cmpComponent.Export VBA_TMP_FOLDER & "" & fileName in memory. 
should be as it is defined in the excel version
    VBA_TMP_FOLDER & "\\\\" & .....


2. It lacks a function call to notify export is done at the end of ExportModules function as opposed to the same functuion in the excel version script.

To test the solution of both issues replace this function at  EXP-SDFER-FDFEWF-CANSEFEF-ACCSDEFEF-DWWCSWEF.vbs :

```

Public Function ExportModules()

    ' Declare variables
    Dim bExport, app, cmpComponent, fileName
    Dim errMessage, appName

    appName = "Access"

    ' Create temp folder for VBA files
    Call TestForDir(VBA_TMP_FOLDER)

    ' Get the Access application object
    Set app = GetObject(DATA_BASE_PATH)

    ' Iterate through all VBA components in the active project
    For Each cmpComponent In app.VBE.ActiveVBProject.VBComponents

        ' Check if the component should be exported
        bExport = True
        fileName = cmpComponent.Name

        ' Concatenation the correct filename for export
        Select Case cmpComponent.Type
            Case 2
                fileName = fileName & ".cls"
            Case 100
                fileName = fileName & ".cls"
            Case 1
                fileName = fileName & ".bas"
            Case Else
                bExport = False
        End Select

        ' Export the component to a text file if it should be exported
        On Error Resume Next
            'Bugfix: 
            cmpComponent.Export VBA_TMP_FOLDER & "\" & fileName
     On Error Goto 0

        ' If an error occurred, log the error message
        If Err.Number <> 0 Then
            errMessage = "Error exporting VBA component: " & Err.Description
            Call LogMessages(0, LOG_FILE_EXPORT, errMessage)
        End If

    Next

    ' Log a message that the export is finished
    Call LogMessages(1, LOG_FILE_EXPORT, "Export VBA files from " & appName & " to " & VBA_TMP_FOLDER & " finished")
    'Bugfix: inserted missing call to notify file monitor routine to copy exported files.
    Call finishExportModules()
End Function
```

**Comment by MichielfromNL** [comment]
Had the same issue.
Is this extension still maintained?
